### PR TITLE
Fix formatting errors

### PR DESCRIPTION
**Pull request by Andre601**
There where some minor formatting errors in https://www.mkdocs.org/user-guide/writing-your-docs/ which I corrected now.

### CODE REVIEW

File: docs/user-guide/writing-your-docs.md [hunk 1]
```diff
@@ -26,8 +26,7 @@ extensions may be used for your Markdown source files: `markdown`, `mdown`,
 `mkdn`, `mkd`, `md`. All Markdown files included in your documentation
 directory will be rendered in the built site regardless of any settings.
 
-!!! note:
-
+!!! note
```
Comment (waylan) [inline docs/user-guide/writing-your-docs.md:29]: The blank line is not a formatting error. Instead the blank line is a matter of style. I am a strong believer in requiring a bank line between each block level item. Therefore, I insist that a blank line remain here. Perhaps the style guide should make this more clear.

File: docs/user-guide/writing-your-docs.md [hunk 2]
```diff
@@ -447,7 +446,6 @@ The first blank line ends all meta-data for the document. Therefore, the first
 line of a document must not be blank.
 
 !!! note
-
```
Comment (waylan) [inline docs/user-guide/writing-your-docs.md:450]: Again, the blank line is intentional.

File: docs/user-guide/writing-your-docs.md [hunk 3]
```diff
@@ -76,8 +75,8 @@ nested URLs, like so:
 
 Any files which are not identified as Markdown files (by their file extension)
 within the [documentation directory](configuration.md#docs_dir) are copied by
-MkDocs to the built site unaltered. See [how to link to images and media]
-(#linking_to_images_and_media) below for details.
+MkDocs to the built site unaltered. See [how to link to images and media](#linking_to_images_and_media)
+below for details.
```
Comment (oprypin) [inline docs/user-guide/writing-your-docs.md:79]: This syntax is valid and doesn't break anything on the site, it's fine to keep as is

**Review by oprypin — COMMENTED** [review]
I think this PR can be closed because all 3 of the changes are either disputed or fine as is.

**Pull request closed without merging**